### PR TITLE
add example for adding a menu option

### PR DESCRIPTION
**Pull request by PhillippOhlandt**
I think it's clear enough.

### CODE REVIEW

File: twig/custom_backend_site.twig [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    quaerat quisquam sunt tempora voluptas voluptate voluptatum?</p><p>Aliquam maiores sequi ut! Aliquid animi
+    architecto blanditiis consequuntur ducimus expedita, inventore ipsum iure labore laudantium perspiciatis saepe
+    similique sint tempora tempore tenetur voluptates? Incidunt minima quidem quos rem vitae.</p><p>Dolore facere, vel.
+    Accusamus architecto assumenda atque, consequuntur cupiditate quasi. At laudantium rem repudiandae suscipit
+    temporibus. Aperiam at blanditiis dolores, dolorum est, et explicabo fugiat harum illo ipsa totam voluptas?</p><p>
+    Alias blanditiis culpa dolorem, doloremque eveniet in itaque possimus quasi ratione repellendus suscipit voluptate.
+    Aliquam architecto distinctio eveniet, facere laborum libero, minus mollitia odit placeat quod repudiandae vitae,
+    voluptas voluptatum.</p><p>Amet consequatur culpa cumque delectus, eaque excepturi fuga fugiat, fugit maiores maxime
+    minus natus necessitatibus, nemo obcaecati odio omnis qui quibusdam quod rem voluptatem! Doloremque id nisi non
+    omnis recusandae!</p>
```
Comment (GwendolenLynch) [inline twig/custom_backend_site.twig:30]: What… the…

Comment (PhillippOhlandt) [inline twig/custom_backend_site.twig:30]: Just to fill all the white space :)

**Comment by PhillippOhlandt** [comment]
Wait, forgot something.

**Comment by PhillippOhlandt** [comment]
:koala: ?

**Comment by GwendolenLynch** [comment]
Le sigh… :stuck_out_tongue_winking_eye:

**Comment by PhillippOhlandt** [comment]
He is alive!

**Pull request merged**